shell/platform/windows/text_input_manager.cc

```diff
@@ -164,7 +164,7 @@ std::optional<std::u16string> TextInputManager::GetString(int type) const {
 }
 
 void TextInputManager::MoveImeWindow(HIMC imm_context) {
-  if (GetFocus() != window_handle_ || !ime_active_) {
+  if (GetFocus() != window_handle_) {
```

Comment:
It looks like other callbacks also have the `!ime_active_` check. Could you shed some light on why this check needs to be removed?